cmd/kube-apiserver/app/options/validation.go

```diff
@@ -45,6 +45,15 @@ func validateServiceNodePort(options *ServerRunOptions) []error {
 	return errors
 }
 
+func validateBootstrapTokenAuthFlags(options *ServerRunOptions) []error {
```

Comment:
this isn't needed... if you use the old flag, you'll get a deprecation warning. in two releases when we remove it, you'll get an error then.